server/sonar-server/src/main/java/org/sonar/server/component/ws/ComponentDtoToWsComponent.java

```diff
@@ -61,6 +61,7 @@ private ComponentDtoToWsComponent() {
       .setKey(dto.getKey())
       .setName(dto.name())
       .setQualifier(dto.qualifier());
+    // TODO add pull request id to the response when it's available in ComponentDto
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fbranch%2Ffirst-class-pull-request&issues=AWGuRf3CaHWQ5zPschp2&open=AWGuRf3CaHWQ5zPschp2)

[ISSUE_KEY]: AWGuRf3CaHWQ5zPschp2